src/Std/Time/Notation/Spec.lean

```diff
@@ -0,0 +1,113 @@
+/-
+Copyright (c) 2024 Lean FRO, LLC. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sofia Rodrigues
+-/
+prelude
+import Std.Time.Date
+import Std.Time.Time
+import Std.Time.Zoned
+import Std.Time.DateTime
+import Std.Time.Format.Basic
+
+namespace Std
+namespace Time
+open Lean Parser Command Std
+
+private def convertText : Text → MacroM (TSyntax `term)
+  | .short => `(Std.Time.Text.short)
+  | .full => `(Std.Time.Text.full)
+  | .narrow => `(Std.Time.Text.narrow)
```

Comment:
Should these be `Lean.Quote` instances?

Reply:
I tried once but I saw that I would need a function called `Unhygienic.run` that I think that is inside `Lean`. AFAIK I cannot import lean things on std because it would lead to some dependency order problems.